src/test/resources/config/application-artemis.yml

```diff
@@ -34,6 +34,8 @@ artemis:
     version-control:
         user: fake-username-vc # The git user. This is required for build agents
         password: fake-password # The git password. This is required for build agents
+        build-agent-git-username: buildjob_user # Required for https access to localvc
```

Comment:
do we really need these values here as well? it should be enough to provide them to `core` nodes (based on `localvc` profile) and to buildagent nodes (based on `buildagent` profile)

Reply:
Yes, these are needed for the server tests. (this config file is just used for tests)